### ISSUE TITLE
make sure we're using the smallest portion of lodash possible

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1265]
@hamilton noticed in  https://github.com/iodide-project/iodide/pull/1250#discussion_r240007725 that we're not importing lodash optimally.

according to this: https://medium.com/@martin_hotell/tree-shake-lodash-with-webpack-jest-and-typescript-2734fa13b5cd
-- webpack should be mostly tree shakeable

> UPDATE August 2017:
> 
> Webpack ( Tobias Koppers ) is rockin’ hard and with the new pure-module feature, lodash will be tree-shake able by default, so no sub paths import will be needed.
> 
> → `import {get,pick} from 'lodash-es'` will just work! more info :

i don't know if we'd need lodash-es or regular lodash, but investigating that is part of this issue

figuring this out and correcting the imports should be an easy fix

**Comment by iFlameing** [comment]
@bcolloran  I found some interesting post regarding the lodash optimization we can use babel-plugin-lodash or lodash-es and setting the "sideeffect" : false in package.json. one of the post I liked it can be found in this [link](https://nolanlawson.com/2018/03/20/smaller-lodash-bundles-with-webpack-and-babel/). what you say do we use plugin or lodash-es module any suggestions.

**Comment by bcolloran** [comment]
hmm lodash-es sounds like it might do what we want. @hamilton do you have thoughts?

**Comment by hamilton** [comment]
yep, we should just be using `loadash-es` which tracks directly to `lodash`.
